src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/MethodConfigurationComponent.scala

```diff
@@ -140,6 +142,28 @@ trait MethodConfigurationComponent {
       }
     }
 
+    object DeleteMethodConfiguration extends RawSqlQuery {
+      val driver: JdbcDriver = MethodConfigurationComponent.this.driver
+
+      def deleteAction(workspaceId: UUID) = {
+        val tables: Seq[SQLActionBuilder] = Seq(sql"METHOD_CONFIG_INPUT", sql"METHOD_CONFIG_OUTPUT",
+          sql"METHOD_CONFIG_PREREQ")
+
+        def baseQuery(table: SQLActionBuilder) = {
+          concatSqlActions(sql"delete t from ", table,
```

Comment:
Can make this simpler by using string literal interpolation instead of using `concatSqlActions`.  Using `$` in a sql-string will SQL-parameterize the value but using `#$` instead will simply insert the string.

Here's how that applies here, noting that `table` is now a String:
```
sql"""delete t from #$table as t inner join METHOD_CONFIG as mc on mc.id=t.method_config_id
                 where mc.workspace_id=${workspaceId}"""
```